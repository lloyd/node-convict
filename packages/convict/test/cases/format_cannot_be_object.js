'use strict';

exports.conf = {
  // default
  none: {
    default: true,
    format: {}
  }
};
