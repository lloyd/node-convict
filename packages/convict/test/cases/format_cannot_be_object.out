none: uses an invalid format, it must be a format name, a function, an array or a known format type (actual: "[object Object]")